### DISCUSSION TITLE
podman-compose how to support mknod: '/dev/loop0': Operation not permitted

### DISCUSSION THREAD

**Discussion by LinglingXiang** [containers/podman#20300] [General]
i am trying to change from docker-compose to podman-compose, i meet two issues

podman exec --interactive --env PULP_SERVER_DATEBASE_SEEDS=pulp-db:27017 --env PULP_SERVER_MESSAGING_URL=tcp://pulp-message:5672 --env PULP_SERVER_MESSAGING_TRANSPORT=rabbitmq --env PULP_SERVER_TASKS_BROKER_URL=amqp://guest:guest@pulp-message// --env PULP_AUTORUN_GENERATE_KEYS=1 --env PULP_CDNDISTRIBUTOR_CDN_HOST=pulp-httpd --env PULP_CDNDISTRIBUTOR_CDN_SSH_LOGIN=cdn --env PULP_AUTORUN_SSHD=1 pub-test_pulp-httpd_1 bash -c mkdir -p /media/iso && devnum=$(losetup -f|sed -e 's/\/dev\/loop//g') && if [ ! -e /dev/loop$devnum ]; then mknod -m660 /dev/loop$devnum b 7 $devnum; fi
mknod: '/dev/loop0': Operation not permitted

**Comment by vrothberg** [comment]
Please reach out to https://github.com/containers/podman-compose with podman-compose specific questions.
